File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_game_tournament_manage_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusGameTournamentManageForm:
    """棋牌游戏赛事管理表单"""
    status: str = None
    sign_up_start_at: str = None
    sign_up_end_at: str = None
    start_at: str = None
    end_at: str = None
    winner_user_id: int = None
    winner_club_id: int = None

File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusGameTournamentManageForm.kt
package com.sdkwork.backend

data class PlusGameTournamentManageForm(
    val status: String? = null,
    val signUpStartAt: String? = null,
    val signUpEndAt: String? = null,
    val startAt: String? = null,
    val endAt: String? = null,
    val winnerUserId: Int? = null,
    val winnerClubId: Int? = null
)
